README.md

```diff
@@ -47,6 +47,9 @@ While we need various naming conventions, this causes problems such as:
 
 To combat these problems, Arachne Digital is trying to promote standardisation where possible, assigning our own names only when a viable alternative does not exist, and dropping our name when one does.
 
+## What are the long term plans for Spindle?
+Check out our roadmap for Spindle [here](https://github.com/arachne-threat-intel/spindle/blob/Community-docs/ROADMAP.md).
```

Comment:
You should leave this as

```
[here](ROADMAP.md).
```

because

```
[here](https://github.com/arachne-threat-intel/spindle/blob/Community-docs/ROADMAP.md).
```
refers to this branch `Community-docs` which will be deleted post this PR being merged. You know you are adding `ROADMAP.md` to the top-level directory so might as well leave the link as just the filename.

Reply:
My bad, lets leave this!

Reply:
Thanks - sorry for the phrasing of 'leave as is', I meant change it